index.js

```diff
@@ -211,14 +210,14 @@ RedisClient.prototype.do_auth = function () {
     self.send_anyway = true;
     self.send_command("auth", [this.auth_pass], function (err, res) {
         if (err) {
-            if (err.toString().match("LOADING")) {
+            if (/LOADING/.test(err.toString())) {
```

Comment:
two thoughts here: can we not use `err.message` rather than `err.toString()`?

also, wouldn't all of these be better off using indexOf? `err.message.indexOf('string') !== -1`